CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_twilio v0.4.0
+
+## Features
+- Added the ability to disable the `USAGE_RECORD` source table and its downstream fields via variable `using_twilio_usage_record` (default `true`). Refer to the [README](https://github.com/fivetran/dbt_twilio?tab=readme-ov-file#step-4-enablingdisabling-models) for more details.
```

Comment:
Should we call out that this variable can be disabled in `twilio__account_overview` and `total_account_spend` will now be removed if this variable is disabled?

Reply:
Good point, added that language!